### PR TITLE
Add ci for python unittests no docker

### PR DESCRIPTION
**Pull request by fpdotmonkey**
changed the unit test cicd to not use docker because I looked into doing docker-in-docker and docker compose, and it seems like a giant pita that we can just skip by running everything directly on whatever machine github gives us.  It's not like we're concerned about the external machine state anyway since the tests run in a venv.

### CODE REVIEW

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -10,5 +10,8 @@ jobs:
       - name: Checkout GitHub workspace
         uses: actions/checkout@v2
 
-      - name: Install python, activate venv, install dependencies, & run tests all in docker container
-        run: sudo docker run --rm -v $PWD:/src -w "/src" ubuntu:latest bash -c "apt update && apt -y upgrade && apt install -y python3-pip python3-venv && ./setup.sh && source ./venv/bin/activate && ./runtests.sh"
+      - name: Install python, activate venv, install dependencies, & run tests
+        run: ./setup.sh
+        run: . venv/bin/activate
+        run: ./runtests.sh
+        run: deactivate
```
Comment (capsulecorplab) [inline .github/workflows/unittests.yml:17]: There's a syntax error in line 14-17. See https://github.com/tetrabiodistributed/project-tetra-display/actions/runs/198822222
Try,
```
run: |
  ./setup.sh
  . venv/bin/activate
  ./runtests.sh
  deactivate
```

Comment (capsulecorplab) [reply .github/workflows/unittests.yml:17]: also, since there's a `deactivate` added in `runtests.sh`, would it be redundant to also add it here?

Comment (fpdotmonkey) [reply .github/workflows/unittests.yml:17]: Fair point yeah

**Comment by fpdotmonkey** [comment]
To check that the tests pass, does it just look at the exit code of the `runtests.sh` script?  Because if so, then I need to make it so that script has a non-zero exit code on fails.

**Comment by capsulecorplab** [comment]
According to the GitHub Actions documentation, a non-zero exit code should result in a failed status check. https://docs.github.com/en/actions/creating-actions/setting-exit-codes-for-actions

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Ventilator Splitter Display
 ===========================
 
+![testing badge](https://github.com/tetrabiodistributed/project-tetra-display/workflows/tests/badge.svg)
```
Comment (capsulecorplab) [inline README.md:4]: Nice

**Pull request merged**